aptos-move/framework/aptos-framework/sources/staking_proxy.spec.move

```diff
@@ -25,14 +26,54 @@ spec aptos_framework::staking_proxy {
     }
 
     spec set_staking_contract_operator(owner: &signer, old_operator: address, new_operator: address) {
-        use aptos_framework::staking_contract::{Store};
-        use aptos_framework::simple_map;
         // TODO: Verify timeout and can't verify `staking_contract::switch_operator`.
+        pragma verify = true;
         pragma aborts_if_is_partial;
```

Comment:
Could you leave a comment indicating why it is still partial?

Reply:
This function related to switch_operator, which contained a vector enum while loop and can't not be verified. Meanwhile it will cause an timeout